compiler_gym/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+################################################################################
+# Autogenerated by build_tools/bazel_to_cmake/bazel_to_cmake.py from           #
+# ../CompilerGym/compiler_gym/BUILD                                            #
+#                                                                              #
+# Use cmake_cmake_extra_content from iree/build_defs.oss.bzl to add arbitrary   #
+# CMake-only content.                                                          #
+#                                                                              #
+# To disable autogeneration for this file entirely, delete this header.        #
+################################################################################
```

Comment:
If the plan is to keep CMake as a parallel build to Bazel, then I suggest that these files not be checked-in. Of course, if replacing Bazel then that is a different matter :-)

Cheers,
Chris

Reply:
This file was auto-generated, but further edits have been made on top of it.